### DISCUSSION TITLE
Select proper COM port for your machine.

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#43] [Q&A] [answered]
Just reading the [flashing instructions](
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10) (since my ESP32 is due to be delivered today!) and was a little confused by the "Select proper COM port for your machine."

How do I know what is the proper COM port?

**Comment by darthcloud** [comment, marked as answer]
The drop box will show all available COM ports. If you only got one serial device plugin you will have only one choice.

Otherwise try them all until one work :)

**Reply by vaderag** [reply]
Thanks. It just arrived and I'm trying this - only get COM3 but it's not detecting anything so assuming that's not the right one 
![image](https://user-images.githubusercontent.com/17175988/123097526-233e5500-d428-11eb-8cba-ae6c70ad356b.png)

In device manager I get the following:
![image](https://user-images.githubusercontent.com/17175988/123097657-3fda8d00-d428-11eb-8ab1-34079814e85f.png)

But not seeing a USB Serial Convertor... 
Any ideas?

**Reply by darthcloud** [reply]
install the CP210x driver:
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/establish-serial-connection.html

**Reply by vaderag** [reply]
Awesome - thanks
Think that worked!!

![image](https://user-images.githubusercontent.com/17175988/123104606-01949c00-d42f-11eb-9073-226daef3288d.png)
